src/main/kotlin/no/nav/familie/ef/sak/behandlingsflyt/steg/StegService.kt

```diff
@@ -188,11 +188,9 @@ class StegService(
         saksbehandlerIdent: String,
         behandlingSteg: BehandlingSteg<T>
     ) {
+        utførBehandlingsvalidering(behandlingSteg, saksbehandling)
```

Comment:
Hadde det vært ennå tydligere for saksbehandler hvis man i `validerGyldigTilstand` sjekket for 
```kotlin
brukerfeilHvis (stegType != BESLUTTE_VEDTAK && saksbehandling.steg.kommerEtter(BESLUTTE_VEDTAK)) {
            "Behandlingen er allerede besluttet"
        }
```

plus flyttet tilgangssjekken til etter `validerGyldigTilstand` ? 

Eller er feilmeldingen tilstrekkelig tydelig som det er endret til nå? 👀

Reply:
Enig - bedre feilmelding, men flytter den heller inn i validering på BehandlingSteg<BeslutteVedtakDto>
Må stokke om på rekkefølgen i utførBehandlingsvalidering, men tenker det egentlig er bedre at spesiell validering kommer før generell.